doc/command_ref.rst

```diff
@@ -894,6 +894,7 @@ Mark Command
 Module Command
 ---------------
 
+Overview about Modularity is available at :ref:`man page dnf.modularity(7) <modularity-label>`.
```

Comment:
It should read:
> Overview of Modularity

To be grammatically correct, although
> Modularity overview

Is the simpler and more straightforward way to say it. I've added the _"General"_ adjective in my previous suggestion to make clearer what the reader will find there.

Reply:
@lukash Thanks, changed